test/unit/host_ansible_role_test.rb

```diff
@@ -17,4 +17,25 @@ class HostAnsibleRoleTest < ActiveSupport::TestCase
     end
     should validate_uniqueness_of(:ansible_role_id).scoped_to(:host_id)
   end
+
+  describe 'auditing' do
+    test 'audit creation and deletion of host ansible roles' do
+      host = FactoryBot.create(:host)
+      ansible_role = FactoryBot.create(:ansible_role)
+      host_ansible_role = FactoryBot.create(:host_ansible_role, :with_auditing, host_id: host.id, ansible_role_id: ansible_role.id)
+
+      assert_not host_ansible_role.audits.blank?
```

Comment:
IMO, it would be clearer to ensure that each test verifies only one audit record is created. This helps confirm that the auditing is working as expected.